vendors/ti/boards/cc3220_launchpad/ports/wifi/iot_wifi.c

```diff
@@ -899,10 +899,6 @@ WIFIReturnCode_t WIFI_GetIPInfo( WIFIIPConfiguration_t * xIPConfig )
 {
     WIFIReturnCode_t xRetVal = eWiFiSuccess;
     int16_t sRetCode;
-
-    configASSERT( xIPConfig != NULL );
-    memset(xIPConfig, 0, sizeof( WIFIIPConfiguration_t ) );
-
```

Comment:
I think we should probably keep the memset should we not?

Reply:
I suppose it doesn't hurt to keep it, though API doesn't specifically mention this is necessary

Reply:
Thanks. Done in latest commit